### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by nperez31**
Update the game over message so people know how to play again!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,51 +14,45 @@ _Collaborate and work together on GitHub._
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 3 >>>
+  Start this step by acknowledging the previous step.
```
Comment (nperez97) [inline README.md:18]: Looks good

**Review by nperez31 — COMMENTED** [review]
Looks good

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (nperez31) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again!</h2>
```

File: README.md [hunk 3]
```diff
@@ -14,51 +14,22 @@ _Collaborate and work together on GitHub._
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 6 >>>
+  Start this step by acknowledging the previous step.
```
Comment (nperez31) [inline README.md:18]: check

File: README.md [hunk 4]
```diff
@@ -14,51 +14,22 @@ _Collaborate and work together on GitHub._
...
+  Define terms and link to docs.github.com.
 -->
 
-## Welcome
+## Step 6: Merge your pull request
```
Comment (nperez31) [inline README.md:22]: test

**Pull request merged**